src/sentry/integrations/jira_server/webhooks.py

```diff
@@ -51,23 +51,29 @@ def dispatch(self, request: Request, *args, **kwargs) -> Response:
         return super().dispatch(request, *args, **kwargs)
 
     def post(self, request: Request, token, *args, **kwargs) -> Response:
+        extra = {}
         try:
             integration = get_integration_from_token(token)
+            extra["integration_id"] = integration.id
         except ValueError as err:
-            logger.info("token-validation-error", extra={"token": token, "error": str(err)})
+            extra.update({"token": token, "error": str(err)})
+            logger.info("token-validation-error", extra=extra)
+            logger.exception("Invalid token.")
```

Comment:
Reporting the issue to Sentry. It allows customer support and triage person to find events for customer issues.